crypto/ct/ct_policy.c

```diff
@@ -25,6 +32,8 @@ CT_POLICY_EVAL_CTX *CT_POLICY_EVAL_CTX_new(void)
         return NULL;
     }
 
+    // time(NULL) shouldn't ever fail, so don't bother checking for -1.
+    ctx->epoch_time_in_ms = (time(NULL) + SCT_CLOCK_DRIFT_TOLERANCE) * 1000;
```

Comment:
This will fail on 32-bit system *today*. Because all involved numbers are 32-bit integers. One of them has to be converted to uint64_t prior multiplication.